test.js

```diff
@@ -0,0 +1,51 @@
+const sinon = require('sinon');
+sinon.behavior = {}; // backwards-compat for referee shenanigans
```

Comment:
There's a bunch of shenanigans going on in this file. One of them is the fact `referee` is dated and hasn't kept up with `sinon` changes. Older `sinon` doesn't get along with `webpack` because older `sinon` likes playing around in globals, or something like that. Also, `sinon-as-promised` is no longer needed with the latest `sinon`, but `referee` requires it. 🔥

Other shenanigans are that `electron-webpack` returns a promise for _reasons_, and `mocha` doesn't care for that, so I dug into it's CLI a bit. I went along with the promise shenanigans for build consistency, but I think later I'll write a minimal `webpack.test.config.js` file, to keep things simple again. But I still need this file anyway for now to get `referee` to play nicely.

Reply:
Let's get rid of referee completely and just go with a combination of `mocha`, `sinon` and `power-assert`. I'm very happy with that setup in various other projects I've been working on this year.

Reply:
@nerdbeere Shenanigans are out, `power-assert` is in! Do you mind having a look and making sure I changed everything over correctly?